tests/_utils/query_tso

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Maybe `run_cdc_cli_tso_query` is better?

Reply:
good